CHANGELOG.rst

```diff
@@ -5,13 +5,22 @@
 
 Renku ``0.37.0`` allows users to pause sessions and later resume them exactly where they left off.
 
+Mind that the previous auto-save mechanism, based on GitLab branches, is phased out. Auto-save branches
+can be safely removed from GitLab repositories.
```

Comment:
this isn't really true - users might have actual saved state there. We need to write up instructions on what they should do with those to be sure they don't lose work. cc @lokijuhy

Reply:
You are right. Better to update that part since the statement is misleading.

What I meant is that branches won't be considered by RenkuLab anymore. There might still be relevant content there -- Tbf I'm not sure anyone will actually go through autosaves to check what's there, but this can be communicated better anyway